features/old/accounts/invoices.feature

```diff
@@ -36,3 +36,41 @@ Feature: Show invoices from account's page (#16015909)
     And I follow "2011-01-00000001"
     # TODO: Properly check if order is opened
     Then I should see "Add"
+
+  Scenario: Display years of invoices when there are invoices
+    Given a provider is logged in on 1st January 2011
+    And a buyer "zoidberg" signed up to provider "foo.3scale.localhost"
+    And the provider is charging its buyers in prepaid mode
+    And an invoice of buyer "zoidberg" for January, 2011
+    And an invoice of buyer "zoidberg" for January, 2014 on 1st January 2014 (without jobs)
+    When I go to the invoices issued by me
+    Then the years selector should have the following options:
+      |      |
+      | 2014 |
+      | 2013 |
+      | 2012 |
+      | 2011 |
+
+  Scenario: Display the current year when there are no invoices
+    Given a provider is logged in on 1st January 2011
+    And the provider is charging its buyers in prepaid mode
+    And I go to the invoices issued by me
+    Then the years selector should have the following options:
+      |      |
+      | 2011 |
+
+  Scenario: Display only years belonging to the current provider
+    Given a provider is logged in on 1st January 2010
+    Given a buyer "zoidberg" signed up to provider "foo.3scale.localhost"
+    And the provider is charging its buyers in prepaid mode
+    And an invoice of buyer "zoidberg" for January, 2010
+
+    Given a provider "boo.3scale.localhost"
+    Given a buyer "zudio" signed up to provider "boo.3scale.localhost"
+    And the provider is charging its buyers in postpaid mode
+    And an invoice of buyer "zudio" for January, 2011
+
+    And I go to the invoices issued by me
+    Then the years selector should have the following options:
```

Comment:
Empty lines inside scenarios are discouraged: it usually means is has too many step or it is written in a complex way. Some common steps could be extracted into a Background and others could be combined into a new step.

This scenario for should communicate better what it tests. This is a suggestion of what I would do IF NOT under /old

```suggestion
  Scenario: Display only years belonging to the current provider
    Given a buyer "zoidberg" of a provider "foo.3scale.localhost"
    And an invoice of buyer "zoidberg" for January, 2010
    And a buyer "zudio" of another provider "boo.3scale.localhost"
    And an invoice of buyer "zudio" for January, 2011
    When an admin logs in to provider "foo.3scale.localhost"
    And go to the invoices
    Then invoices can be selected from the following years:
```